app/javascript/react/components/App.js

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import UserIndex from './UserIndex'
 import UserShowContainer from './UserShowContainer'
+import AboutJewelryBox from './AboutJewelryBox'
 
 export const App = () => {
   return (
     <BrowserRouter>
       <Switch>
         <Route exact path="/" component={UserIndex} />
         <Route exact path="/users" component={UserIndex} />
+        <Route exact path="/about" component={AboutJewelryBox} />
```

Comment:
I'd suggest grouping similar routes together, such as listing `about` first, then grouping all of the `users` routes